server/src/main/kotlin/com/hockey/elo/elotracker/user/model/UserLoginSubmission.kt

```diff
@@ -1,6 +1,11 @@
 package com.hockey.elo.elotracker.user.model
 
+import javax.validation.constraints.NotBlank
+
 data class UserLoginSubmission(
-    val name: String,
-    val rfid: String
-)
\ No newline at end of file
+        @NotBlank
+        val name: String,
+
+        @NotBlank
```

Comment:
should we allow users to login using not an RFID? Like email or phone number? 

That would allow users who aren't Solstice Emp to also play!